custom/up_nrhm/reports/block_level_month_report.py

```diff
@@ -79,7 +79,8 @@ def avg(values, idx=None):
             config['enddate'] = (
                 config['startdate'] + relativedelta(months=1) - relativedelta(days=1)
             ).replace(hour=23, minute=59, second=59)
-            data.append(ASHAFacilitatorsData(config).data)
+            for index, row_data in ASHAFacilitatorsData(config).data.items():
```

Comment:
Based on the columns in this report I think it should only ever return a single row

Reply:
Still getting a hold of how these reports are working so really don't understand why that would be and comment on that.

Ideally i think all reports should assume receiving back multiple rows with/without group by where the key would be index or group by columns and if a report expects only a single row, it should raise an error if multiple were returned.

Reply:
all the columns are aggregate columns (sums or counts) so the SQL should only return a single row